trulens_eval/trulens_eval/guardrails/base.py

```diff
@@ -0,0 +1,56 @@
+from concurrent.futures import wait
+from trulens_eval.feedback import Feedback
+from trulens_eval.utils.containers import first
+from trulens_eval.utils.containers import second
+from trulens_eval.utils.threading import ThreadPoolExecutor
+from functools import wraps
+
+class context_filter:
+    def __init__(self, feedback: Feedback, threshold: float):
```

Comment:
can we move `__init__()` below the docstring? some completion tools/parsers don't pick it up otherwise